src/main.js

```diff
@@ -15,34 +15,76 @@
 var Util = require('./util.js');
 var WebVRPolyfill = require('./webvr-polyfill.js');
 
+var RENAMED_BOOL_CONFIG_OPTIONS = {
```

Comment:
personally, I've spent way too much time chasing down regressions, intentional breakage without warning. I can remove, but I know it's causing _a lot_ of headache for folks today. and it's becoming a problem for some of us early adopters.